<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCProductSettingFrame</class>
 <widget class="QFrame" name="ICHCProductSettingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>27</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>36</number>
   </property>
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>27</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Product</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="productLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Wait Mold Opened Limit Time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="waitTimeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Position When Arm Down</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="posComboBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Vertical Down</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horizontal Down</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Alarm Times</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Times</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="alarmTimesEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
